### ISSUE TITLE
Rule Request: Multiline method call indentation

### ISSUE THREAD

**Issue by Drarok** [realm/SwiftLint#2931]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### New rule request

I'd like to enforce a multiline indent style that doesn't appear to be possible with the current available options. I don't know what the name should be, but the idea is to enforce only a single indent level increase when splitting a method call over multiple lines.

#### Non Triggering Examples

```swift
foo(bar: "bar",
    baz: "baz"
)
```

```swift
foo(
    bar: "bar", baz: "baz"
)
```

```swift
foo(
    bar: "bar",
    baz: "baz"
)
```

```swift
CustomButton("Request Password Reset", style: .primary, action: model.requestPasswordResetCall)
```

```swift
CustomButton("Sign in", style: .secondary, action: {
    self.viewState = .signIn
})
```

#### Triggering Examples (assuming 4 spaces indent)

```swift
longer_named_foo(bar: "bar",
                 baz: "baz")
```

```swift
longer_named_foo(
        bar: "bar",
        baz: "baz")
```

```swift
longer_named_foo(
        bar: "bar",
        baz: "baz"
)
```

I don't know what configuration this might need, but it should probably be opt-in.

What does everyone else think?

**Comment by Drarok** [comment]
Closing a ticket for inactivity is a sure-fire way to alienate people from contributing, if you ask me…
